app/models/label/LabelTable.scala

```diff
@@ -266,12 +266,13 @@ object LabelTable {
       """SELECT lb1.label_id, lb1.gsv_panorama_id, lp.heading, lp.pitch, lp.zoom, lp.canvas_x, lp.canvas_y,
         |       lp.canvas_width, lp.canvas_height, lb1.audit_task_id, u.user_id, u.username, lb1.time_created,
         |       lb_big.label_type, lb_big.label_type_desc, lb_big.severity, lb_big.temp_problem, lb_big.description
+        |
```

Comment:
Was there any reason to change the two lines in this query? If not, could you change them back? It is best to not modify any lines of code unless you are purposefully making changes to them. One reason is that I modify this query in another PR I was working on to fix the indentation and style, but if you also make changes to it, I will have to resolve merge conflicts for no reason :wink: 

So before I run `git add` on any file, I do a `git diff` on it to see if I made any changes on accident, and I make sure to revert any accidental changes.